README.md

```diff
@@ -1,95 +1,251 @@
+window = create_mmgui(exec_app=False)
 
-  See [CONTRIBUTING](CONTRIBUTING.md) for local setup instructions.
+# customize the app or do other setup here
+# mmcore = window.mmcore  # access the CMMCorePlus used by the GUI
 
-  You can download the latest nightly bundled applications [here](https://nightly.link/pymmcore-plus/pymmcore-gui/workflows/bundle/main)
+QApplication.instance().exec()  # Start the Qt event loop
+```
 
-</details>
+If you already have a `CMMCorePlus` instance that you want the GUI to use, you
+can pass it to `create_mmgui(mmcore=my_core)`. By default the GUI will first
+check if there is a global singleton (`CMMCorePlus.instance()`), and if not, it
+will create a new instance.
 
-## Goals (and non-goals) of unification
+## Prior Work
 
-### Goals
+This project builds upon several prior efforts in the Python microscopy
+community:
 
-- Provide a napari-independent GUI for controlling micro-manager via
-  pymmcore-plus (i.e. pure python micro-manager control).  We'd like to have a
-  primary application that we can point interested parties to (rather than
-  having to describe all the related efforts and explain how to compose
-  pymmcore-widgets directly).
-- Avoid duplicate efforts.  While independent related projects are excellent in
-  that they allow rapid exploration and experimentation, we'd like to be able to
-  share the results of these efforts.  In some ways that is done via
-  pymmcore-widgets, but all of the application level stuff (persistence of
-  settings, complex layouts, coordination of data saving, viewing & processing)
-  is explicitly not part of pymmcore widgets.
-- Establish patterns for persistence and application state.
+- [**napari-micromanager**](https://github.com/pymmcore-plus/napari-micromanager):
+  A plugin using Napari as the viewer and pymmcore-widgets for UI. It
+  demonstrated Micro-Manager control in a python GUI. While it doesn't receive
+  active updates, it remains usable for those who prefer a napari-based
+  workflow.
+- [**micromanager-gui**](https://github.com/fdrgsp/micromanager-gui) by Federico
+  Gasparoli.
+- [**pymmcore-plus-sandbox**](https://github.com/gselzer/pymmcore-plus-sandbox)
+  by Gabe Selzer.
 
-### Non-Goals
+Lessons from these prototypes (and others at labs like LEB-EPFL) have influenced
+pymmcore-gui’s design.
 
-- Working on a shared application is *not* meant to discourage independent
-  experimentation and repositories.  (One of the real strengths in doing this
-  all in python is the ease of creating custom widgets and GUIs!).  One possible
-  pattern would be forks & branches off of a main central repository.
+By unifying ideas from these projects, pymmcore-gui aims to provide a single,
+officially supported application. Our design goal is a user experience familiar
+to Micro-Manager users (for example, one prototype mimicked the MMStudio layout
+￼), while taking advantage of Python’s flexibility and the growing ecosystem of
```

Comment:
Found another one of these unrendered characters 🤔

Reply:
sorry, fixing!